internal/gencli/cmd_file.go

```diff
@@ -276,32 +276,15 @@ var {{$methodCmdVar}} = &cobra.Command{
 		{{ end }}
 		{{ end }}
 		{{ else if and .Paged (not .IsLRO ) }}
-		// get requested page
-		var items []interface{}
-		data := make(map[string]interface{})
-
-		// PageSize could be an integer with a specific precision.
-		// Doing standard i := 0; i < PageSize; i++ creates i as
-		// an int, creating a potential type mismatch. 
-		for i := {{ .InputMessageVar }}.PageSize; i > 0; i-- {
-			item, err := iter.Next()
-			if err == iterator.Done {
-				err = nil
-				break
-			} else if err != nil {
-				return err
-			}
-
-			items = append(items, item)
+		// populate iterator with a page
+		_, iterErr := iter.Next()
```

Comment:
Can we change this to `_, err = iter.Next()` rather than creating a new error variable?

Reply:
Wasn't sure if `err` was always declared earlier, hence `listErr` to be safe. Could change to `err = ` instead

Reply:
`err` is a [named return value](https://github.com/googleapis/gapic-generator-go/blob/master/internal/gencli/cmd_file.go#L116) of the encapsulating function. So we are good to use it here!